src/ert/config/ert_config.py

```diff
@@ -372,12 +372,32 @@ def read_site_config(cls) -> ConfigDict:
 
     @classmethod
     def read_user_config(cls, user_config_file: str) -> ConfigDict:
-        site_config = cls.read_site_config()
-        return lark_parse(
-            file=user_config_file,
-            schema=init_user_config_schema(),
-            site_config=site_config,
-        )
+        return lark_parse(user_config_file, schema=init_user_config_schema())
+
+    @classmethod
+    def read_user_config_and_apply_site_config(
+        cls, user_config_file: str
+    ) -> ConfigDict:
+        site_config_dict = cls.read_site_config()
+        user_config_dict = cls.read_user_config(user_config_file)
+
+        for keyword, value in site_config_dict.items():
+            if keyword == "QUEUE_OPTION":
+                filtered_queue_options = []
+                for queue_option in value:
+                    option_name = queue_option[1]
+                    if option_name in user_config_dict:
+                        continue
+                    filtered_queue_options.append(queue_option)
+                user_config_dict["QUEUE_OPTION"] = (
```

Comment:
I'm confused by this one. I thought we want to do it other way around; ie. overwriting the site_config with user config?

Reply:
Resolved by discussion :)